GooglePlayInstant/LoadingScreen/LoadingBar.cs

```diff
@@ -13,172 +13,94 @@
 // limitations under the License.
 
 using System.Collections;
-#if UNITY_EDITOR
-using UnityEditor;
-#endif
 using UnityEngine;
-using UnityEngine.UI;
 
 namespace GooglePlayInstant.LoadingScreen
 {
     /// <summary>
-    /// Class that encapsulates the creation and update of a Loading Bar component in the editor and during the game's
-    /// runtime.
+    /// Presents download progress to the user
     /// </summary>
-    public static class LoadingBar
+    [ExecuteInEditMode]
+    public class LoadingBar : MonoBehaviour
     {
-        // TODO: revisit these arbitrarily chosen values
-        /// <summary>
-        /// Percentage of the loading bar allocated to the the asset bundle downloading process.
-        /// </summary>
-        public const float AssetBundleDownloadMaxWidthPercentage = .8f;
-
-        /// <summary>
-        /// Percentage of the loading bar allocated to the the scene loading process.
-        /// </summary>
-        public const float SceneLoadingMaxWidthPercentage = 1 - AssetBundleDownloadMaxWidthPercentage;
-
-        // Loading bar fill's padding against the loading bar outline.
-        private const int LoadingBarFillPadding = 17;
+        public float OutlineWidth = 10f;
+        public float InnerBorderWidth = 10f;
 
-        // Loading bar size as a proportion of the screen size. Adjust if needed.
-        private const float LoadingBarWidthProportion = 0.7f;
-        private const float LoadingBarHeightProportion = 0.02f;
+        [Tooltip("If true, this object's RectTransform will update to adjust the outline and border width")]
+        public bool ResizeAutomatically = true;
 
-        // Loading bar placement as a proportion of the screen size relative to the bottom left corner. Adjust if needed.
-        private const float LoadingBarPositionX = 0.5f;
-        private const float LoadingBarPositionY = 0.3f;
+        [Range(0f, 1f)] public float Progress = 0.25f;
```

Comment:
Maybe add a [Tooltip] like "Asset Bundle download and install progress. The value set in the Editor is ignored at runtime."